reactor-netty-http/src/main/java/reactor/netty/http/server/HttpServer.java

```diff
@@ -210,6 +210,21 @@ public final HttpServer forwarded(boolean forwardedEnabled) {
 		return dup;
 	}
 
+	/**
```

Comment:
Can we change the method `forwarded(boolean forwardedEnabled)` to the following:

```
public final HttpServer forwarded(boolean forwardedEnabled) {
	if (forwardedEnabled &&
			configuration().forwardedHeaderHandler == DefaultHttpForwardedHeaderHandler.INSTANCE) {
		return this;
	}
	HttpServer dup = duplicate();
	dup.configuration().forwardedHeaderHandler = DefaultHttpForwardedHeaderHandler.INSTANCE;
	return dup;
}
```